candi/openapi/node_group.yaml

```diff
@@ -162,6 +162,9 @@ spec:
               required:
                 - nodeType
               properties:
+                quickShutdown:
+                  type: boolean
+                  description: Lowers machine drain timeout to 5 minutes.
```

Comment:
I think this parameter should be in the cloudInstances key, because it affects only machines under mcm management. This parameter is not useful for static nodes.

Reply:
Fixed.